.github/workflows/ci.yml

```diff
@@ -102,7 +102,7 @@ jobs:
         if [[ ${{ matrix.db-type }} == 'pgsql' ]]; then export PGSQL_DSN='pgsql://postgres:postgres@127.0.0.1/phinx'; fi
 
         if [[ ${{ matrix.prefer-lowest != 'prefer-lowest' }} ]]; then
-          export CODECOVERAGE=1 && vendor/bin/phpunit --verbose --coverage-clover=coverage.xml
+          export CODECOVERAGE=1 && vendor/bin/phpunit --coverage-clover=coverage.xml
```

Comment:
I think the biggest change would be no longer printing out the skipped tests, but I think that's actually preferable as they were essentially just noise for the most part and something to annoyingly scroll past to see actual test errors.

If we wanted to bring back the behavior, can use the new `--display-skipped` flag or `displayDetailsOnSkippedTests` XML option.